### ISSUE TITLE
KIF fails to find accessibility labels on iOS 10

### ISSUE THREAD

**Issue by razgon** [kif-framework/KIF#936]
Hi!
KIF fails to find accessibility labels on iOS 10.2.1(14D10) I'm wondering is it a problem with iOS or project setup. Issue seems like an access restriction, but what is root cause?
`2017-01-05 16:33:57.648977 Application[1072:227184] [User Defaults] Failed to write value for key ApplicationAccessibilityEnabled in CFPrefsPlistSource<0x17011c050> (Domain: com.apple.Accessibility, User: kCFPreferencesCurrentUser, ByHost: No, Container: (null)): setting preferences outside an application's container requires user-preference-write or file-write-data sandbox access, switching to read-only
2017-01-05 16:33:57.660906 Application[1072:227135] [User Defaults] attempt to set 1 for key in AutomationEnabled in read-only (due to a previous failed write) preferences domain CFPrefsPlistSource<0x17011c050> (Domain: com.apple.Accessibility, User: kCFPreferencesCurrentUser, ByHost: No, Container: (null))
2017-01-05 16:33:57.662777 Application[1072:227135] [User Defaults] attempt to set 1 for key in ApplicationAccessibilityEnabled in read-only (due to a previous failed write) preferences domain CFPrefsPlistSource<0x17011c050> (Domain: com.apple.Accessibility, User: kCFPreferencesCurrentUser, ByHost: No, Container: (null))
2017-01-05 16:33:57.663501 Application[1072:227135] [User Defaults] attempt to set 1 for key in AccessibilityEnabled in read-only (due to a previous failed write) preferences domain CFPrefsPlistSource<0x17011c050> (Domain: com.apple.Accessibility, User: kCFPreferencesCurrentUser, ByHost: No, Container: (null))`

**Comment by mikelupo** [comment]
Hi @razgon. Thank you for your report.

I have Xcode Version 8.2.1 (8C1002) and with an iPad Retina 10.2 Simulator (14C89), I am able to run my KIF tests.

Need a bit more background info.
Simulator or hardware? If hardware, I assume 10.2.1 beta 1?
What version of Xcode?

**Comment by johnslay** [comment]
I seem to be having issues with tapRow:inTableViewWithAccessibilityIdentifier. KIF will execute the method just fine, but it actually does not click on the cell, so the screen does not change, thus failing to find accessibility labels following the call. All my other table views are working just fine. This one in particular started failing once I started running KIF tests on iOS 10.

**Comment by imsrc21** [comment]
I am facing same issue here (**Assertions: The step timed out after 10.00 seconds: Failed to find accessibility element with the label "Red"**)

XCode: 8.3.2
iOS: 10.3.1

**Comment by justinseanmartin** [comment]
@imsrc21 - Responded to your issue in #970.

@razgon - If your issue persists, please reactivate this issue with more information. Thanks!

**Comment by readwrite123455** [comment]
failed: caught "KIFFailureException", "The step timed out after 60.00 seconds: Failed to find accessibility element with the label "Action - Trip""

**Comment by readwrite123455** [comment]
Hi Justin ,
I am facing with issues now.
Any help would be appreciated

**Comment by mikelupo** [comment]
Post an example app on github that exhibits the problem. Link it here so we can pull it and reproduce.
